### ISSUE TITLE
Investigate 'Corresponding Author'

### ISSUE THREAD

**Issue by jesusbagpuss** [digital-york/wrro-pure#12]
Is this information represented in the MODS record?

If the corresponding author is a York author, this should be preferred as the 'contact_email' for Request-a-Copy.

**Comment by jesusbagpuss** [comment]
Also: check https://github.com/digital-york/wrro-pure/blob/master/xslt/mods_to_eprints_york_for_upload.xslt#L75-L79 to see which author is selected.
Does it generate multiple `contact_email` elements, or just the first/last (or something else)?

**Comment by jesusbagpuss** [comment]
Corresponding Author doesn't seem to be available in the MODS representation (Pure v5.24.1).

**Comment by jesusbagpuss** [comment]
This data is now available to crosswalk.
WR approach is to add a new field to the EPrint record to store this data.

The MODS representation is:
```xml
<mods:name type="personal" xlink:type="simple">
  <mods:role>
    <mods:roleTerm type="text" authority="pure/role">author</mods:roleTerm>
  </mods:role>
  <mods:namePart type="given">John</mods:namePart>
  <mods:namePart type="family">Salter</mods:namePart>
  <mods:role>
    <mods:roleTerm type="text" authority="pure/email">someone@somewhere.com</mods:roleTerm>
  </mods:role>
  <mods:role>
    <mods:roleTerm type="text" authority="pure/role">correspondingAuthor</mods:roleTerm>
  </mods:role>
</mods:name>
```

NB The email address may not be present for external authors.
